### ISSUE TITLE
Missing TypeScript and Python code snippets for installation updates

### ISSUE THREAD

**Issue by heath-hamilton** [MicrosoftDocs/msteams-docs#3142]
Since the feature is in preview, these may be coming soon, but we need TypeScript and Python code snippets for the [install update event](https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/subscribe-to-conversation-events?tabs=dotnet#install-update-event) section to fix the docs build warning stating you must use the same set of tabs throughout a page. In this case, we need the tabs to match the other tab sets like this:

|C# |TypeScript | JSON |Python |

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0a6d00a4-dd28-33c4-002e-2e8e7c653ce2
* Version Independent ID: 0856927a-25f4-b1e0-5f1b-d55dc61796f3
* Content: [Conversation events - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/subscribe-to-conversation-events?tabs=dotnet)
* Content Source: [msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md)
* Product: **msteams**
* GitHub Login: @WashingtonKayaker
* Microsoft Alias: **anclear**

**Comment by Wajeed-msft** [comment]
Code snippets:

JavaScript:
```
async onInstallationUpdateActivity(context: TurnContext) {
        var activity = context.activity.action;
        if(activity == "Add") {
            await context.sendActivity(MessageFactory.text("Added"));
        }
        else {
            await context.sendActivity(MessageFactory.text("Uninstalled"));
        }
    }
```


Python: 
```
async def on_installation_update(self, turn_context: TurnContext):
   if turn_context.activity.action == "add":   
       await turn_context.send_activity(MessageFactory.text("Added"))
   else:
       await turn_context.send_activity(MessageFactory.text("Uninstalled"))
```

**Comment by surbhigupta12** [comment]
Closing this issue as it is resolved In #4996
